app/assets/javascripts/miq_application.js

```diff
@@ -1695,11 +1695,48 @@ function chartData(type, data, data2) {
       _.isObject(data.axis.y.tick) &&
       _.isObject(data.axis.y.tick.format) &&
       data.axis.y.tick.format.function) {
+
     var format = data.axis.y.tick.format;
+    max = _.max(getChartColumnDataValues(data.data.columns));
```

Comment:
This should have a `var` too. (CC may warn about it, but ignore that, double var is way less bad then forgetting about it and having an accidental global)